### ISSUE TITLE
Ability to allow breakpoints at illegal locations

### ISSUE THREAD

**Issue by hdalsania** [apache/daffodil-vscode#848]
It will let you put a breakpoint on a line that is not an element, but it doesn't stop there. Could it somehow indicate that no breakpoint there is possible?

Source of this ticket is approval VOTE [email](https://lists.apache.org/thread/0gcddqd3mhys5179t8zgkpp51l9oc6nv) from @mbeckerle for v1.3.1-rc1 on 08/29/2023

**Comment by arosien** [comment]
@mbeckerle DAP models a "verified" boolean for breakpoints, settable by the debugger. We could have the backend parse the set of files mentioned in the breakpoints and validate the locations. This can be done synchronously (when the `SetBreakpoint` request is received) or asynchronously (via the `BreakpointEvent` event).

**Comment by michael-hoke** [comment]
In setBreakpoints, there is a line `case (bp, i) => new Types.Breakpoint(i, true, bp.line, "")`. The hard-coded `true` corresponds to whether the breakpoint is verified.

Questions:
- Where do we draw the line between front end and back end determining validity? Front end can handle comments, empty lines, lines outside of root element, etc.. Back end can handle schema contents?
- Doing the front end work will require parsing of the whole schema to figure out if we're in a multiline comment. This will be exacerbated if we end up doing schema content validation on the front end.

**Comment by scholarsmate** [comment]
This is medium priority, but _may_ slip depending on developer availability.

**Comment by hdalsania** [comment]
I agree this is a medium priority and does require proper assessment, I moving this out of v1.6.0 scope.
